modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -1,55 +1,31 @@
 = DataWeave System Properties
-
-
-DataWeave supports several system properties. To use these properties in application development and deployments, see xref:mule-runtime::mule-app-properties-system.adoc[System Properties].
-
+DataWeave supports several system properties. To use these properties in application development and deployments, see xref:mule-runtime:mule-app-properties-system.adoc[System Properties].
 [%header%autowidth.spread,cols=",a"]
 |===
 |Property |Description
-
-//com.mulesoft.dw.buffer.memory.monitoring//
-|`com.mulesoft.dw.buffer.memory.monitoring`|
-_Experimental:_ When set to `true`, logs a message each time a slot from the memory pool is taken or released. The behavior is useful for troubleshooting.
-`com.mulesoft.dw.buffer.memory.monitoring` supports an _experimental feature_ that is subject to change or removal from future versions of DataWeave.
-
-* Type: `Boolean`
-* Default: `false`
-
-
-//com.mulesoft.dw.buffered_char_sequence.enabled
+//com.mulesoft.dw.buffered_char_sequence.enabled//
 |`com.mulesoft.dw.buffered_char_sequence.enabled`|
-Controls whether an internal `BufferedCharSequence` improvement is enabled or disabled. When this property is set to `true`, DataWeave avoids loading large fields (greater than 1.5M) into memory. These fields are read in chunks. This feature reduces performance to prevent an out-of-memory error.
-
-
+Controls whether an internal BufferedCharSequence improvement is enabled or disabled. When this property is set to `true`, DataWeaves avoid load large fields (bigger than 1.5M) into memory. This fields are read by chunk. This feature introduces a performance reduction preventing an Out of Memory error.
```

Comment:
```suggestion
Controls whether an internal `BufferedCharSequence` improvement is enabled or disabled. When this property is set to `true`, DataWeave avoids loading large fields (greater than 1.5M) into memory. These fields are read in chunks. This feature reduces performance to prevent an out-of-memory error.
```

Reply:
added to master